api/v1alpha4/types.go

```diff
@@ -129,17 +129,28 @@ type VirtualMachineCloneSpec struct {
 	// virtual machine is cloned.
 	// +optional
 	MemoryMiB int64 `json:"memoryMiB,omitempty"`
-	// DiskGiB is the size of a virtual machine's disk, in GiB.
+	// PrimaryDisk holds the specs of the primary disk of the virtual machine.
+	// Defaults to the eponymous property value in the template from which the
+	// virtual machine is cloned.
+	// +optional
+	PrimaryDisk Disk `json:"primaryDisk,omitempty"`
```

Comment:
given that we're releasing v1alpha4 this week, I would sugget making this change additive so that we're able to release it in `v0.8.x`

Reply:
I'm sorry, I do not follow. Could you explain a bit more what needs to be done for making an additive change?

Is it just keeping the DiskGiB or I just post a PR for changing just this field for now to PrimaryDisk and then add AdditionalDisks later?

Reply:
Sounds like the idea is to leave DiskGiB for backward compatibility